src/Client.php

```diff
@@ -360,14 +360,18 @@ public function waitForJobCompletion(string $queryJobId, int $maxWaitSeconds = 3
     {
         $startTime = time();
 
+        $tries = 0;
         while (time() - $startTime < $maxWaitSeconds) {
             $status = $this->getJobStatus($queryJobId);
 
             if (in_array($status['status'], ['completed', 'failed', 'canceled'], true)) {
                 return $status;
             }
 
-            sleep(1);
+            // 10, 20, 40, 80, 160, 320, 640, 1000ms (max)
+            $waitMilliseconds = min(pow(2, $tries)*10, 1000);
```

Comment:
nevím, možná by to tu stačilo 100, 200, 400, 800, 1000m
nebo začínat trochu dál s tou řadou 110, 120, 140, 180, 260, 320, 740, 1000ms

Reply:
ok, přičteme k tomu vždy 50, tak to začne na 60ms :)